### PR TITLE
MSPSDS-286 accessible autocomplete

### PR DESCRIPTION
**Pull request by AnthonyPengelly**

### CODE REVIEW

File: app/views/investigations/assign.html.erb [hunk 1]
```diff
@@ -1,33 +1,24 @@
...
 
-<h1 class="heading-xlarge">Update assignee</h1>
 <h4 class="heading-medium">Current assignee</h4>
 <p><%= @investigation.assignee.nil? ? "Unassigned" : @investigation.assignee.email %></p>
 
 <%= form_with(url: update_assignee_investigation_path(@investigation), local: true) do |form| %>
   <div class="form-group">
     <%= form.label :email, :class => "form-label" %>
     <%= form.select :email,
+                    # When there are many users, we should consider loading them with ajax using the autocomplete
```
Comment (broder) [inline app/views/investigations/assign.html.erb]: Should we do this now?

File: package.json [hunk 2]
```diff
@@ -13,6 +13,7 @@
   },
   "license": "MIT",
   "dependencies": {
+    "accessible-autocomplete": "^1.6.1",
```
Comment (broder) [inline package.json]: Should we put this on "Accessibility Discovery of the Day"?

File: app/views/investigations/assign.html.erb (hunk 1 above)
Comment (AnthonyPengelly) [reply app/views/investigations/assign.html.erb]: I'm happy to pick this up as my next task. I thought due to time restrictions it's better to get this through the door, and then we can make it ajax-y as another task. (I had a quick go, but it didn't Just Work, so I moved on)

Comment (AnthonyPengelly) [reply app/views/investigations/assign.html.erb]: I'm happy to pick this up as my next task. I thought due to time restrictions it's better to get this through the door, and then we can make it ajax-y as another task. (I had a quick go, but it didn't Just Work, so I moved on)

Comment (AnthonyPengelly) [reply app/views/investigations/assign.html.erb]: Also, due to disabling prefix search, your autocomplete experience will be strictly worse once we move it to ajax 😛 though I think it's worth doing anyway

Comment (broder) [reply app/views/investigations/assign.html.erb]: Don't worry about doing it now - I was hoping that it would Just Work 😐

**Pull request merged**